index.template.html

```diff
@@ -3916,6 +3925,191 @@ <h5>Inferencing of Personally Identifiable Information Privacy Risk</h5>
 
   </section>
 
+  <section id="thing-templates" class="normative">
+  <h1>Thing Model</h1>
+      
+  <p class="note">
+    The following section has its origin in [[wot-thing-description]], Annex C. Here Thing Description 
+    Template is renamed to <a>Thing Model</a>, but keeps the same intention.
+    For this version of the specification, <a>Thing Model</a> and its model features (e.g., extendions, obligations, placeholder) are formal 
+    introduced.
+  </p>
+
+    <p>
+    A <em>Thing Model</em> is a special kind of a <a>Thing Description</a> with less restriction as 
+    driven in <a href="#sec-vocabulary-definition"></a> and <a href="#sec-td-serialization"></a>. Thing Model samples can 
+    also be seen as a subset of the <a>Thing Descriptions</a> information model, which may not contain, 
+    for example, concrete protocol and security information.
+    </p>
+    <p>
+    The Thing Model enables:
+    <ul>
+    <li> onboard and manage of multiple <a>Thing</a> models, e.g., by a cloud service. </li>
+    <li> simulate of devices/<a>Things</a> that have not yet been developed.</li>
+    <li> develop common applications across devices from different manufacturers that share a common <a>Thing</a> model.</li>
+    <li> combining multiple models into a <a>Thing</a>.
+    <li> implementation support of a concrete <a>Thing</a>.
+    </ul>
+    </p>
+    <p>
+    The Thing Model is a logical description of the interface and possible interaction with <a>Thing's</a>
+    <a>Properties</a>, <a>Actions</a>, and <a>Events</a>, however it does not contain <a>Thing</a> instance-specific
+    information, such as security information and concrete protocol usage, or even a serial number and GPS location.
```

Comment:
I think TMs *should* be *allowed* to include security schemes.   Note that security schemes don't declare instance-specific information anyhow.   They *might*, however, have URLs (like token servers) that might need to be omitted or parameterized (with templates) although in a lot of cases these might also be given (a TM might easily define a set of instances all of whom share a specific token server).   Simplest thing here is just to remove the "for instance" phrase.  Actually, I also often want to specify the protocol so the best you can say here is "such as concrete fully-specified URLs"

Reply:
I can also mention this.

Reply:
done